try {
    new Error('something wrong');
} catch (error){
    console.log(error);
}